File: Modul 2/py/PRAK205-2310817110007-MuhammadDaffaMusyafa.py
import math
tinggi = input("")
sisi_miring= input("")

alas = round(math.sqrt(int(sisi_miring) * int(sisi_miring) - int(tinggi) * int(tinggi)))

keliling = round(int(tinggi) + int(sisi_miring) + int(alas))

luas = round((int(tinggi) * int(alas)) / 2)

print(f"Alas ={alas} cm")
print(f"keliling ={keliling} cm")
print(f"luas = {luas} cm^2")

File: Modul 2/C/PRAK205-2310817110007-MuhammadDaffaMusyafa.c
#include <stdio.h>
#include <math.h>

int main(){
    int a, b, c, luas, keliling;
    scanf("%d %d", &a, &b);

    c = sqrt((b * b) - (a * a));
    keliling = a + b + c;
    luas = (a * c) / 2;



    printf("Alas: %d cm\n", c);
    printf("Tinggi: %d cm\n", a);
    printf("keliling: %d cm\n", keliling);
    printf("luas: %d cm^2\n", luas);

    return 0;
}